pkg/transform/oauth/github_test.go

```diff
@@ -17,17 +17,25 @@ import (
 
 func TestTransformMasterConfigGithub(t *testing.T) {
 	file := "testdata/github-test-master-config.yaml"
-	content, _ := ioutil.ReadFile(file)
+
+	content, err := ioutil.ReadFile(file)
+	require.NoError(t, err)
 
 	serializer := k8sjson.NewYAMLSerializer(k8sjson.DefaultMetaFactory, scheme.Scheme, scheme.Scheme)
 	var masterV3 configv1.MasterConfig
-	_, _, _ = serializer.Decode(content, nil, &masterV3)
+
+	_, _, err = serializer.Decode(content, nil, &masterV3)
```

Comment:
Thank you for fixing all of these in the tests. I made a bit of a mess copy/pasting them into all the tests. I'll working on creating a helper function and try to get rid of a lot of it altogether later.